### PR TITLE
[NCL-2539] Configuration loaded multiple times

### PR DESCRIPTION
**Pull request by thescouser89**
We instead use the Application Scope PNC to inject the already loaded configuration.

This is an updated PR to #1297 . I have removed the `@Inject` in the enum, and instead provide the configuration to the method requiring `configuration` via the method parameter.

### CODE REVIEW

File: openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/OpenshiftStartedEnvironment.java [hunk 1]
```diff
@@ -85,6 +87,9 @@
     private Route route;
     private Service sshService;
 
+    @Inject
+    private Configuration configuration;
```
Comment (matejonnet) [inline openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/OpenshiftStartedEnvironment.java]: OpenshiftStartedEnvironment is also instantited using new, thus not managed by the CDI.

Comment (thescouser89) [inline openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/OpenshiftStartedEnvironment.java]: Changed! Thanks for catching this again!

I'm not really happy with my fix though.

**Comment by thescouser89** [comment]
Woah sorry about dimissing the review. I don't know how to just say "It's been resolved" on Github.

**Comment by thescouser89** [comment]
Closed! Updated the PR in #1408

**Pull request closed without merging**